eap/eap71-tx-recovery-s2i.json

```diff
@@ -39,21 +39,28 @@
             "displayName": "Git Repository URL",
             "description": "Git source URI for application",
             "name": "SOURCE_REPOSITORY_URL",
-            "value": "https://github.com/jboss-developer/jboss-eap-quickstarts",
+            "value": "https://github.com/jboss-openshift/openshift-quickstarts.git",
             "required": true
         },
         {
             "displayName": "Git Reference",
             "description": "Git branch/tag reference",
             "name": "SOURCE_REPOSITORY_REF",
-            "value": "7.1.0.GA",
+            "value": "1.3",
```

Comment:
Is this the correct tag?  I believe some changes went in recently that modified this quickstart, so it works with the new version of tx recovery added through CLOUD-2261.

Reply:
@rcernich Ah, I wasn't aware of those. Looks like they're still just in master.

@ochaloup WDYT about this, is it ok to default these, or should we wait for a 1.4 tag and use that?

Reply:
I can create a 1.4 branch if you like.  That would help separate things, for cases where CLOUD-2261 hasn't been released in an image yet.

Reply:
@rcernich I think we may want to wait for @ochaloup for that, in case he's still working on stuff. For now, lets push this to S23 I think, its done, but lower priority for merge IMO.

Reply:
@luck3y @rcernich I had a short PTO so I'm a bit late on response.

Regarding the [CLOUD-2261|https://issues.jboss.org/browse/CLOUD-2261]. The changes were already merged to the master of cct_module. Based on the changes in [CLOUD-2261|https://issues.jboss.org/browse/CLOUD-2261] there was needed a fix in quickstarts [CLOUD-2760|https://issues.jboss.org/browse/CLOUD-2760]. That's already merged in the quickstarts master.
I'm sorry for that but I'm a bit unsure about release process so I would ask for your guidance to say what branch or tag is needed to be used, or what's the right time to merge this.

Up to that there is issue [CLOUD-2768|https://issues.jboss.org/browse/CLOUD-2768] which I started to investigate about but it has lower priority. I would like to merge the both jta quickstarts of eap6/eap7 to one directory. There will be still two separate `tepmplate.json` files thus this PR does not need to be blocked in this manner.

Reply:
Thanks, @ochaloup. I think in that case we should tag version 1.4 of the OS quickstarts and I can update this PR for that.

Reply:
@luck3y, do I need to create a tag for quickstarts so we can merge this?

Reply:
@rcernich Yeah, we need a 1.4, IMO.

Reply:
branch 1.4 created in jboss-openshift/openshift-quickstarts